core/src/main/scala/com/spotify/featran/converters/DefaultTransform.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.featran.converters
+
+import com.spotify.featran.transformers.{Identity, Transformer}
+
+/**
+ * Default Type Class used by the from generator for Case Class Conversions
+ */
+trait DefaultTransform[T] {
+  def apply(featureName: String): Transformer[T, _, _]
+}
+
+case object IdentityDefault extends DefaultTransform[Double] {
```

Comment:
@richwhitjr here I would probably do the following:

```
object DefaultTransformers {
    implicit val identityDefault: DefaultTransform[Double] = ...
}
```

i think it gives more room to add default transforms and it's in a defined "context"

wdyt?